packages/scene-composer/src/components/StateManager.tsx

```diff
@@ -116,15 +116,16 @@ const StateManager: React.FC<SceneComposerInternalProps> = ({
     setEditorConfig({
       operationMode: config.mode,
       uriModifier: standardUriModifier,
-      valueDataBindingProvider,
+      // Currently only support single data binding provider. Need to update this code to support multiple.
+      valueDataBindingProvider: valueDataBindingProviders?.TwinMakerEntityProperty,
```

Comment:
if there are consumers using `SceneComposerInternal` component directly instead of `SceneViewer` component, then this change from `valueDataBindingProvider` (object) into `valueDataBindingProviders` (dict of objects) will be a breaking change. but I highly doubt this would be the case since implementing a valueDataBindingProvider is hard and `SceneComposerInternal` is meant to be an internal used component...

Reply:
so the type of the provider is also changed a bit. I will mark this commit as breaking change and do major version bump then